src/CTBase.jl

```diff
@@ -17,7 +17,7 @@ using DocStringExtensions
 using ForwardDiff: jacobian, gradient, ForwardDiff # automatic differentiation
 using Interpolations: linear_interpolation, Line, Interpolations # for default interpolation
 using Reexport
-@reexport using MLStyle # pattern matching 
+@reexport using MLStyle # pattern matching
```

Comment:
we should be able not to reexport `MLStyle.SyntaxError` (now renamed to `ParsingError` in `CTBase`)